syntax="proto3";

service Backend {
    rpc load_image (img_path) returns (image);
    rpc predicting (img_path) returns (inferencia);
}

message image {
    string img_content = 1;
    int32 width = 2;
    int32 height = 3;
}

message img_path {
    string path = 1;
}

message inferencia{
    string val_inferencia = 1;
}
